CHANGELOG.md

```diff
@@ -12,9 +12,10 @@
 
 
 ### Bugfixes
--Removes inactive danger areas at EIDW [#562](https://github.com/openscope/openscope/issues/562)
+- Removes inactive danger areas at EIDW [#562](https://github.com/openscope/openscope/issues/562)
```

Comment:
no worries.

It looks like there is also an open invite out there for you to join the organization.  Could you accept that?  Then all future changes can be made in this repo which will avoid the need for a fork and allow our tests to run hen you open a PR.

You can also join us on slack for faster communication http://slack.openscope.co